frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -1,23 +1,580 @@
+            />
+            <TextInput
+              label="Social Security Disability Income"
+              dataKey="person.income.ssdi"
+            />
+            <small className="web-form-input">(SSDI)</small>
+            <TextInput
+              label="Supplemental Security Income"
+              dataKey="person.income.ssi"
+            />
+            <small className="web-form-input">(SSI)</small>
+            <TextInput
+              label="Social Security"
+              dataKey="person.income.socialSecurityOther"
+            />
+            <small className="web-form-input">(Other than SSDI or SSI)</small>
+            <TextInput
+              label="Unemployment benefits"
+              dataKey="person.income.unemploymentBenefits"
+            />
+            <TextInput
+              label="Educational benefits"
+              dataKey="person.income.educationalBenefits"
+            />
+            <small className="web-form-input">
+              (Including grants, loans, cash scholarships, etc.)
+            </small>
+            <TextInput
+              label="Veteran's benefits"
+              dataKey="person.income.veteransBenefits"
+            />
+            <TextInput label="Alimony" dataKey="person.income.alimony" />
+            <TextInput
+              label="Child Support"
+              dataKey="person.income.childSupport"
+            />
+            <TextInput
+              label="Payments from civil litigation "
+              dataKey="person.income.civilPayments"
+            />
+            <TextInput
+              label="Victim restitution"
+              dataKey="person.income.victimRestitution"
+            />
+            <TextInput
+              label="Public assistance"
+              dataKey="person.income.publicAssistance"
+            />
+            <small className="web-form-input">
+              (Including AFDC, FEP, TANF, welfare, etc.)
+            </small>
+            <TextInput
+              label="Financial support from household members"
+              dataKey="person.income.householdFinancialSupport"
+            />
+            <TextInput
+              label="Financial support from non-household members"
+              dataKey="person.income.otherFinancialSupport"
+            />
+            <TextInput
+              label="Trust income"
+              dataKey="person.income.trustIncome"
+            />
+            <TextInput
+              label="Annuity income"
+              dataKey="person.income.annuityIncome"
+            />
+          </>
+        )}
+        <div className="web-form-input">Total:</div>
+        <CheckBox
+          label="I have no income because:"
+          dataKey="person.noGrossMonthlyIncome"
+          name="grossMonthlyIncome"
+        />
+        {data.person.noGrossMonthlyIncome && (
+          <TextArea
+            label="Explain"
+            dataKey="person.noGrossMonthlyIncome.explanation"
```

Comment:
same here about how `person.noGrossMonthlyIncome` is trying to act as both a boolean and an object.